docs/markdown/Using Pants/advanced-target-selection.md

```diff
@@ -31,18 +31,11 @@ To run against another branch, run:
 By default, `--changed-since` will only run over files directly changed. Often, though, you will want to run over any [dependees](doc:project-introspection) of those changed files, meaning any targets that depend on the changed files. Use ` --changed-dependees=direct` or ` --changed-dependees=transitive` for this:
 
 ```bash
-$ ./pants \
+❯ ./pants \
   --changed-since=origin/main \
   --changed-dependees=transitive \
   test
 ```
-[block:callout]
-{
-  "type": "warning",
-  "title": "Using a version control system other than Git?",
-  "body": "Please message us on Slack or open a GitHub issue (see [Community](doc:getting-help)). We would be happy to look into adding support for your VCS, such as helping you with a PR to add support."
-}
-[/block]
```

Comment:
Unrelated change. My thinking is that we have too much content in our docs that is distracting. It slows readers down.

Do we think many folks are using not-Git?